### PR TITLE
fixed KustoSinkTaskTest

### PR DESCRIPTION
**Pull request by SanchayGupta1197**
Fixed the issue for KustoSInkTaskTest
-Mocked the validateTableMapping 
- added null check for context while running test

### CODE REVIEW

File: src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java [hunk 1]
```diff
@@ -193,7 +194,7 @@ public TopicIngestionProperties getIngestionProps(String topic) {
         return topicsToIngestionProps.get(topic);
     }
 
-    private void validateTableMappings(KustoSinkConfig config) {
+    public void validateTableMappings(KustoSinkConfig config) {
```
Comment (fahadhasher) [inline src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java:197]:
```suggestion
    void validateTableMappings(KustoSinkConfig config) {
```
This way it's package protected and will be visible to respective test class.

File: src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java [hunk 2]
```diff
@@ -320,7 +321,9 @@ public void start(Map<String, String> props) {
         flushInterval = config.getFlushInterval();
         log.info(String.format("Started KustoSinkTask with target cluster: (%s), source topics: (%s)", 
             url, topicsToIngestionProps.keySet().toString()));
-        open(context.assignment());
+        if(context!=null) {
```
Comment (fahadhasher) [inline src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java:324]:
```suggestion
        // Adding this check to make code testable
        if (context != null) {
```

Comment (SanchayGupta1197) [reply src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java:324]: done

File: src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java (hunk 1 above)
Comment (SanchayGupta1197) [reply src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java]: done

File: src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java (hunk 2 above)
Comment (SanchayGupta1197) [reply src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java:324]: done

**Pull request merged**